### PR TITLE
Make dark mode background darker and links lighter (#795)

### LINKED ISSUE #792: The Links on the Contacts page , help page & settings (manage cookies) don't have enough contrast
**Description**:
## Dark Mode 

Links on manage cookies, help page and  contact page don't have enough contrast to satisfy WCAG 2 AA. Suggested foreground colour change to #2685ff

help_page(darkmode)
<br><br>
contacts_page(darkmode)
<br><br>
cookies(darkmode)
**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done

### LINKED ISSUE #793: The footer links don't have enough contrast
**Description**:

## Dark mode 
Doesn't have enough contrast  to satistfy WCAG 2 AA.Suggested foreground color change to #2685ff .

footer links(darkmode)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done

### PR DESCRIPTION
**Pull request by joelvdavies**
## Description
This fixes contrast issues with some of the URLs on https://github.com/ral-facilities/datagateway/issues/867 and https://github.com/ral-facilities/datagateway/issues/901, as well as fixing https://github.com/ral-facilities/datagateway/issues/898, https://github.com/ral-facilities/datagateway/issues/900, #793, #792. This also makes the dark mode background slightly darker in order to make it easier to make foreground colours compliant with WCAG 2.0.

## Testing instructions

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage

## Agile board tracking
Closes ral-facilities/datagateway#898, Closes ral-facilities/datagateway#900, Closes #793, Closes #792

### CODE REVIEW

**Comment by joelvdavies** [comment]
Here are some images:
image
image

**Review by joshuadkitenge — APPROVED** [review]
Everything works as expected. Wait until after the sprint meeting to merge. Add more pictures showing the different between the original vs your changes please.

**Pull request merged**
